store/src/main.rs

```diff
@@ -39,10 +40,13 @@ async fn main() -> Result<()> {
 /// Sends a gRPC request as specified by `command`.
 ///
 /// The request is sent to the endpoint defined in `config`.
+#[instrument(target = "miden-store", skip_all)]
 async fn query(
     config: StoreTopLevelConfig,
     command: Query,
 ) -> Result<()> {
+    info!(target: COMPONENT, ?config, ?command);
+
```

Comment:
I don't think we need to do logging here. This is just as CLI command intended for testing and if we want to see some output, we should use `println!()` (but I don't think we need to do it here.